internal/tiltfile/skylarkTypes.go

```diff
@@ -159,7 +159,17 @@ func addMount(thread *skylark.Thread, fn *skylark.Builtin, args skylark.Tuple, k
 		return nil, errors.New("internal error: add_docker_image_cmd called on non-dockerImage")
 	}
 
-	image.mounts = append(image.mounts, mount{mountPoint, gitRepo})
+	gr, ok := src.(gitRepo)
```

Comment:
might be slightly cleaner to do a typeswitch

switch gr := gr.(type) {
  case localPath:
  case gitRepo:
  default:
}

Reply:
ah yes good call, forgot about those!